libs/utils/env.py

```diff
@@ -764,7 +764,13 @@ def calibration(self, force=False):
                   force=False and we have not installed rt-app.
         """
 
-        if not force and self._calib:
+        if not force:
```

Comment:
How about installing rt-app when calibration is requested ? AFAIK `install_tools()` does the difference with already-installed tools, so we can let the `__init__` call to `calibration()` not do anything but the user could call `calibration` with `force=True` to install rt-app ? There would have to be some renaming/extra args so that all of this makes sense to the user, but that's the general idea.